src/components/Hero/Hero.style.js

```diff
@@ -0,0 +1,60 @@
+import { css } from '@emotion/react';
+import colors from '../../styles/colors';
+import mainBg from '../../../public/images/main-bg.png';
+import { Lexia } from '../../styles/fonts';
+
+const styles = () => css`
```

Comment:
this is a suggestion:
when we don't need dynamic values the css string doesn't have to be returned by a function:
const styles = css`super-cool-style: 100%;`; will suffice